cfme/tests/infrastructure/test_individual_host_creds.py

```diff
@@ -105,7 +107,9 @@ def _refresh():
                   BZ(1584246, forced_streams=['5.8', '5.9'],
                      unblock=lambda provider: not provider.one_of(VMwareProvider)),
                   BZ(1584261, forced_streams=['5.8'],
-                     unblock=lambda creds: creds == 'default')])
+                     unblock=lambda creds: creds == 'default'),
```

Comment:
**REQUIRED**
Can you please remove obsolete blockers here?

Reply:
@quarckster I'm don't see any obsolete blockers, all of them are NEW except the new one which is already ASSIGNED.